doc/notebooks/jupyter_integration.ipynb

```diff
@@ -75,17 +75,35 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Let's display the DTM of our sample area to ensure all's working\n",
-    "\n",
+    "# Let's see what is in the example database\n",
+    "print(gs.read_command(\"g.list\", type=\"all\"))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
     "# Set computational region to the study area.\n",
-    "gs.parse_command(\"g.region\", raster=\"elevation\", flags=\"pg\")\n",
+    "gs.run_command(\"g.region\", raster=\"elevation\", flags=\"pg\")"
```

Comment:
Should be sufficient to use either p or g flag.

Reply:
With run_command, nothing is printed (as expected), so the `-p` is useless. Either -g and parse_command or nothing. I think nothing is fine here. For reading region to Python we have a better function anyway (`region`).